src/java/com/zimbra/qa/selenium/projects/ajax/tests/calendar/meetings/organizer/singleday/viewinvite/ViewInviteWithDisplayMailPreference.java

```diff
@@ -51,14 +51,21 @@ public ViewInviteWithDisplayMailPreference() throws HarnessException {
 	)
 	public void ViewHTMLInviteWithHTMLPreference_01() throws HarnessException {
 		
+		String fontfamilyData="";

```

Comment:
Minor, please replace String fontfamilyData=""; to String fontFamilyData = ""; Use camelcase variable name,